packages/plugins/plugin-deck/src/components/DeckLayout/Plank.tsx

```diff
@@ -75,8 +75,16 @@ export const Plank = memo(({ entry, layoutParts, part, layoutMode, order, last }
 
   useLayoutEffect(() => {
     if (scrollIntoView === coordinate.entryId) {
-      rootElement.current?.focus({ preventScroll: true });
-      layoutMode === 'deck' && rootElement.current?.scrollIntoView({ behavior: 'smooth', inline: 'center' });
+      // TODO(wittjosiah): When focused on page load, the focus is always visible.
+      //   Forcing focus to something smaller than the plank prevents large focus ring in the interim.
+      const focusable =
+        rootElement.current?.querySelector('input') ||
+        rootElement.current?.querySelector('button') ||
+        rootElement.current;
+      focusable?.focus({ preventScroll: true });
+      layoutMode === 'deck' && focusable?.scrollIntoView({ behavior: 'smooth', inline: 'center' });
```

Comment:
This seems offset by some amount, but we should fix that by implementing a deck-aware method later, this is great for now.